proposals/0417-task-executor-preference.md

```diff
@@ -551,6 +551,48 @@ actor Worker {
 }
 ```
 
+### The `DefaultConcurrentExecutor` executor
+
+This proposal also introduces a way to obtain a reference to the default global concurrent executor which is used by default by all tasks and asynchronous functions unless they require some specific executor. 
+
+```swift
+/// A task executor which enqueues all work on the default global concurrent
+/// thread pool that is used as the default executor for Swift concurrency
+/// tasks.
+public final class DefaultConcurrentExecutor: _TaskExecutor {
+  public static let shared: DefaultConcurrentExecutor
+  
+  public func enqueue(_ job: consuming ExecutorJob)
+  public func asUnownedTaskExecutor()
+}
+
+extension _TaskExecutor where Self == DefaultConcurrentExecutor {
+
+  /// The default executor preference, setting it using ``withTaskExecutor(_:)``
+  /// is equivalent to disabling an existing task executor preference, or
+  /// stating that task now has "no task executor preference".
+  public static var `default`: DefaultConcurrentExecutor {
+    DefaultConcurrentExecutor.shared
+  }
+
+}
+```
+
+This executor does not introduce new functionality to Switft Concurrency per se, as it was always there since the beginning of the concurrency runtime, however it is the first time it is possible to obtain a reference to the global concurrent executor in pure Swift. Generally just creating tasks and calling nonisolated asynchronous functions would automatically enqueue them onto this underlying global threadpool.
+
+This proposal introduces the `DefaultConcurrentExecutor` type in order to be able to "disable" a task executor preference, because setting a task's executor preference to the default executor is equivalent to the task having the default behavior, as if no executor preference was set. This matters particularily wich child tasks, which do want, under any circumstances, to execute on the default executor:
```

Comment:
```suggestion
This proposal introduces the `DefaultConcurrentExecutor` type in order to be able to "disable" a task executor preference, because setting a task's executor preference to the default executor is equivalent to the task having the default behavior, as if no executor preference was set. This matters particularily with child tasks, which do want, under any circumstances, to execute on the default executor:
```